### PR TITLE
[Delivers #85750808] Show embed level when not signed in

### PR DESCRIPTION
**Pull request by wjordan**

### CODE REVIEW

**Comment by laurelfan** [comment]
looks good, though it seems like the "show an embedded level" action is getting to be sufficiently different from the "show the level in the context of admin/level building" action that we're using #show for that it may justify making it its own action.

**Comment by joshlory** [comment]
+1 for adding a new Level#embedded action

File: dashboard/app/controllers/levels_controller.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 class LevelsController < ApplicationController
   include LevelsHelper
   include ActiveSupport::Inflector
-  before_filter :authenticate_user!
+  before_filter :authenticate_user!, :unless => Proc.new {params[:embed]}
```
Comment (laurelfan) [inline dashboard/app/controllers/levels_controller.rb]: oh wait, I think we only want to do the unless for the show action.

(I think the authorize! filter will catch all the cases that would get through as the code is now (except for index) but I think it's safer to limit things as much as we can (esp. if we change something later))

**Comment by wjordan** [comment]
Added the extra restriction suggested by Laurel. I agree refactoring will be helpful here but will defer this further pass until later since it's non-trivial (will involve making a pass on integrated blockly/apps code) and this feature was requested for end of day today.

**Pull request merged**